modules/Attendance/attendance.php

```diff
@@ -75,7 +75,7 @@
 
 
 // define attendance tables, if user is permit to view them
-if (isset($_SESSION[$guid]["username"])) {
+if (isset($session->get("username"))) {
```

Comment:
This is one case where an isset() on the return value for the get method is not ideal. There is a $session->has() method that can be used for this case, which also makes the line more readable.
```suggestion
if ($session->has('username')) {
```